src/assets/style/_elements.scss

```diff
@@ -540,7 +540,7 @@ $gc-small: $form-width - $gc-large - 4;
 
 .cdk-overlay-pane {
   .mat-dialog-container {
-    max-height: 80vh !important;
+    // max-height: 80vh !important;
```

Comment:
why not to remove?

Reply:
wasn't sure if we still need it and we have to fix the height in another way. But yes, let's delete this line 😉

Reply:
done in 924fc3d